crates/next-core/src/app_structure.rs

```diff
@@ -26,6 +26,12 @@ use crate::{
     next_import_map::get_next_package,
 };
 
+// Next.js ignores underscores for routes but you can use %5f to by still serve an underscored
```

Comment:
nit: An extra word showed up here
```suggestion
// Next.js ignores underscores for routes but you can use %5f to still serve an underscored
```